hedera-node/hedera-mono-service/src/test/java/com/hedera/node/app/service/mono/ledger/ids/SeqNoEntityIdSourceTest.java

```diff
@@ -80,7 +80,11 @@ void returnsExpectedAccountId() {
         given(seqNo.getAndIncrement()).willReturn(555L);
 
         // when:
-        AccountID newId = subject.newAccountId(sponsor);
+        AccountID newId = AccountID.newBuilder()
+                .setRealmNum(sponsor.getRealmNum())
+                .setShardNum(sponsor.getShardNum())
```

Comment:
nit: This should be 
```
AccountID.newBuilder()
                .setRealmNum(0)
                .setShardNum(0)
```
Should not matter for the test.

Reply:
The change that you suggested makes the test fail on line 90 `        assertEquals(asAccount("1.2.555"), newId);`

Reply:
Agreed, after our conversation last friday `.setRealmNum(0)` and `.setShardNum(0)` are what I thought we were going with (and the test would be changed to "0.0.555"). Someone remember something different?

Reply:
That sounds right. I will update the test to make it work.